polus-ome-zarr-converter-plugin/README.md

```diff
@@ -0,0 +1,28 @@
+# OME Zarr Converter
+
+This WIPP plugin converts BioFormats supported data types to the OME Zarr file
+format.
+
```

Comment:
Brief description about the OME Zarr file format or links to documentation will be useful.